crates/common/batcher/src/config.rs

```diff
@@ -1,4 +1,4 @@
-use chrono::Duration;
+use time::Duration;
```

Comment:
```suggestion
use std::time::Duration;
```
Or using `time::Duration` was inetentional?

Reply:
In this case using `time::Duration` is intentional due to maths done on it in `batch.rs`